koa-crud/src/pagination/index.ts

```diff
@@ -0,0 +1,41 @@
+import R from 'ramda';
+
+const paginate = (data, first, after) => {
+  if (first < 0) {
+    throw new Error(`Invalid first`);
+  }
+
+  const modifiedData = R.map((node: any) => {
+    return R.assoc('rawCursor', node.cursor.buffer, node);
+  })(data);
```

Comment:
@lafbelmonte  so the `data`'s shape here is this, right?
```typescript
[
  {
     node: {},
     cursor: Buffer
  } 
]
```

In my opinion the data should come from the result of the mongoose model.

Reply:
No sir, that's for adding rawCursor property to the nodes. Reason for this is because the main buffer is nested from the cursor property(mongoose thingy) and it is easier to R.findIndex to just add another property that holds the buffer. Please check my latest commit I removed this function.